cmd/lncli/commands.go

```diff
@@ -2012,6 +2012,34 @@ var cltvLimitFlag = cli.UintFlag{
 		"this payment",
 }
 
+var paymentFlags = []cli.Flag{
```

Comment:
not blocking, but could make this a function to avoid the double appends later on

Reply:
I was looking for a way. Good suggestion, done.